build-system/compile/compile.js

```diff
@@ -298,6 +295,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       jscomp_off: [],
       define,
       hide_warnings_for: hideWarningsFor,
+      debug: !!argv.pseudo_names,
```

Comment:
For a closer mapping of `gulp dist` args to closure compiler args, why not change this to `argv.debug`?

Reply:
id rather not have users have to relearn a new flag